### PR TITLE
Generic front review

### PR DESCRIPTION
**Pull request by abulte**
This a proposition for integrating #91 into this repo.

Main changes:
- config file loaded from an env var `CONFIG_NAME`
- config files format change with nested structure
- stuff not used by ecospheres has been component-sized 
- BouquetStore and TopicStore have been merge
- ChartStore no longer necessary, uses TopicStore or DatasetStore directly
- fix a warning when using `fill` on LineChart @geoffreyaldebert check this is what you wanted :-)

I'm not totally crazy about [my last commit](https://github.com/ecolabdata/ecospheres-front/pull/93/commits/7070941dcc6c2700bea29c269bc293925dfe1116), things are still not done very cleanly IMHO, but maybe better than before this commit? I can remove it.

### CODE REVIEW

**Review by geoffreyaldebert — APPROVED** [review]
💪 thanks all seems good for me

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ This template should help get you started developing with Vue 3 in Vite.
 
 See [Vite Configuration Reference](https://vitejs.dev/config/).
 
+`CONFIG_NAME` env var is used to control which file in `configs/{CONFIG_NAME}/config.yaml` is used as config file. You can set it in the `.env` file(s).
```
Comment (streino) [inline README.md:13]: Niiiice 👍

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -15,31 +17,16 @@ const markdown = new MarkdownIt()
...
-watchEffect(() => {
-  chartStore.getCharts("topics", config.universe_topic_id)
-  if(topics.value) {
-    topicStore.getTopics(topics.value)
-  }
-})
 
+const homepageTitle = config.website.homepage_title
+const homepageSubTitle = config.website.homepage_subtitle
+const topics = config.website.list_highlighted_topics
```
Comment (streino) [inline src/views/HomeView.vue:27]: Nitpick: Le nommage est pas super cohérent: list_highlighted_topics, home_buttons, show_topic_charts
Mais on peut voir plus tard.

File: src/views/bouquets/BouquetDetailView.vue [hunk 3]
```diff
@@ -2,7 +2,7 @@
 import { onMounted, ref, computed } from "vue"
 import { useRoute, useRouter } from "vue-router"
 
-import { useBouquetStore } from "../../store/BouquetStore"
+import { useTopicStore } from "../../store/TopicStore"
```
Comment (streino) [inline src/views/bouquets/BouquetDetailView.vue:5]: Nice too :)

**Review by streino — APPROVED** [review]
Très cool ! Merci pour le boulot !

File: src/views/HomeView.vue (hunk 2 above)
Comment (geoffreyaldebert) [reply src/views/HomeView.vue:27]: on pourrait renommer en `home_topics` du coup peut-être

Comment (abulte) [reply src/views/HomeView.vue:27]: Oui voyons plus tard, ma valeur ajoutée n'est pas énorme là-dessus :-)

**Pull request merged**